src/utils.js

```diff
@@ -13,7 +13,8 @@ export function isUnsafeCallee(node) {
   // For Function we are looking for this: `Function("...")();`
   // A double CallExpression
   return [
-    identifier === "eval" || (identifier === "Function" && node.callee.type === "CallExpression"),
+    (identifier === "eval" && node.callee.type === "Identifier") ||
```

Comment:
Not sure why we need to add `node.callee.type === "Identifier"` here?

Reply:
i have removed this unecenssary part in latest commit (was trying to enforce some side case i was creating in my firsts attempts)